src/modal.rs

```diff
@@ -108,7 +109,7 @@ pub trait Modal: Sized {
     ///
     /// Optionally takes an initialized instance as pre-filled values of this modal (see
     /// [`Self::execute_with_defaults()`] for more info)
-    fn create(defaults: Option<Self>) -> serenity::CreateInteractionResponse<'static>;
+    fn create(defaults: Option<Self>, custom_id: Option<String>) -> serenity::CreateInteractionResponse<'static>;
```

Comment:
Can you remove the Option from the custom_id parameter? I think it's confusing to pretend you can omit the custom_id, because the implementation will unconditionally set a custom_id anyways